scripts/analysis_0/nbconverted/4_create_figs_talk.py

```diff
@@ -0,0 +1,468 @@
+    "Batch_effects",
+    "output",
+    "talk figures",
+    "analysis_0_svcca_overlay_blk.png")
+
+svcca_overlay_long_file = os.path.join(
+    local_dir,
+    "Data",
+    "Batch_effects",
+    "output",
+    "talk figures",
+    "analysis_0_svcca_overlay_long.png")
+
+svcca_overlay_long_blk_file = os.path.join(
+    local_dir,
+    "Data",
+    "Batch_effects",
+    "output",
+    "talk figures",
+    "analysis_0_svcca_overlay_long_blk.png")
+
+
+# In[4]:
+
+
+# Load pickled files
+uncorrected_svcca = pd.read_pickle(similarity_uncorrected_file)
+err_uncorrected_svcca = pd.read_pickle(ci_uncorrected_file)
+corrected_svcca = pd.read_pickle(similarity_corrected_file)
+err_corrected_svcca = pd.read_pickle(ci_corrected_file)
+permuted_score = np.load(permuted_score_file)
+
+
+# In[5]:
+
+
+# Concatenate error bars
+uncorrected_svcca_err = pd.concat([uncorrected_svcca, err_uncorrected_svcca], axis=1)
+corrected_svcca_err = pd.concat([corrected_svcca, err_corrected_svcca], axis=1)
+
+
+# In[6]:
+
+
+# Add group label
+uncorrected_svcca_err['Group'] = 'uncorrected'
+corrected_svcca_err['Group'] = 'corrected'
+
+
+# In[7]:
+
+
+# Concatenate dataframes
+all_svcca = pd.concat([uncorrected_svcca_err, corrected_svcca_err])
+all_svcca
+
+
+# In[8]:
+
+
+# Plot - uncorrected only
+lst_num_experiments = list(all_svcca.index[0:int(len(all_svcca.index)/2)])
+
+threshold = pd.DataFrame(
+    pd.np.tile(
+        permuted_score,
+        (len(lst_num_experiments), 1)),
+    index=lst_num_experiments,
+    columns=['score'])
+
+g = ggplot(all_svcca[all_svcca['Group'] == 'uncorrected'])     + geom_line(all_svcca[all_svcca['Group'] == 'uncorrected'],
+                aes(x=lst_num_experiments, y='score', color='Group'),
+                size=1.5) \
+    + geom_point(aes(x=lst_num_experiments, y='score'), 
+                 color ='darkgrey',
+                size=0.5) \
+    + geom_errorbar(all_svcca[all_svcca['Group'] == 'uncorrected'],
+                  aes(x=lst_num_experiments, ymin='ymin', ymax='ymax'),
+                   color='darkgrey') \
+    + geom_line(threshold, 
+                aes(x=lst_num_experiments, y='score'), 
+                linetype='dashed',
+                size=1,
+                color="darkgrey",
+                show_legend=False) \
+    + labs(x = "Number of Experiments", 
+           y = "Similarity score (SVCCA)", 
+           title = "Similarity across varying numbers of experiments") \
+    + theme(plot_title=element_text(weight='bold'),
+            plot_background=element_rect(fill="white"),
+            panel_background=element_rect(fill="white"),
+            panel_grid_major_x=element_line(color="lightgrey"),
+            panel_grid_major_y=element_line(color="lightgrey"),
+            axis_line=element_line(color="grey"),
+            legend_key=element_rect(fill='white', colour='white')
+           ) \
+    + scale_color_manual(['#b3e5fc']) \
+
+print(g)
+ggsave(plot=g, filename=svcca_uncorrected_file, dpi=300)
```

Comment:
all of these newly color themed plots are ok for the purpose of the talk(s) but you might want to make colors more visible for a paper

Reply:
Any suggestions on colors?

Reply:
shades a couple tones darker for each will do, if simple shift is preferred. i.e. instead of baby blue and blue, dodger blue and navy, for example.